integration/src/test/resources/ldap_jaas.config

```diff
@@ -225,13 +225,13 @@ ldaptive-props {
     ldapUrl="${ldapTestHost}"
     bindDn="${ldapBindDn}"
     bindCredential="${ldapBindCredential}"
-    connectTimeout="8000"
+    connectTimeout="PT8S"
```

Comment:
The ISO period/duration syntax may take some users time to learn, but it's very meaningful once you get the hang of it. No more ambiguity about what the units are (s/ms/min).